com.ntrack.n-track.yaml

```diff
@@ -0,0 +1,89 @@
+app-id: com.ntrack.n-track
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "23.08"
+command: n-TrackStudio
+modules:
+  - name: ntrack
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 16x16.png ${FLATPAK_DEST}/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png
+      - install -Dm644 22x22.png ${FLATPAK_DEST}/share/icons/hicolor/22x22/apps/${FLATPAK_ID}.png
+      - install -Dm644 32x32.png ${FLATPAK_DEST}/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm644 48x48.png ${FLATPAK_DEST}/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm644 64x64.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png
+      - install -Dm644 128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
+      - install -Dm644 256x256.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm644 512x512.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra
+      - mkdir /app/share/vulkan
+      - ln -s /usr/lib/${FLATPAK_ARCH}-linux-gnu/GL/vulkan/implicit_layer.d /app/share/vulkan/implicit_layer.d
+      - ln -s ${FLATPAK_DEST}/extra/opt/n-Track/n-TrackStudio ${FLATPAK_DEST}/bin/n-TrackStudio
+    sources:
+      - type: extra-data
+        filename: n-track_flathub.tar.gz
+        only-arches:
+          - aarch64
+        url: https://ntrack.com/n-track_aarch64_flathub.tar.gz
+        sha256: 1903667ff03b4e6da3a67035ce4d2f955e8feb45409a21d4c9fa6c87c2d8f40c
+        size: 260875147
+      - type: extra-data
+        filename: n-track_flathub.tar.gz
+        only-arches:
+          - x86_64
+        url: https://ntrack.com/n-track_x86_64_flathub.tar.gz
+        sha256: 83705cf0a63e1f473201d641b8c9fb6a0bd4eb5895cf465e78fbdedcec4fc40a
+        size: 385409446
+      - type: file
+        path: com.ntrack.n-track.metainfo.xml
+      - type: file
+        path: com.ntrack.n-track.desktop
+      - type: file
+        path: 16x16.png
+      - type: file
+        path: 22x22.png
+      - type: file
+        path: 32x32.png
+      - type: file
+        path: 48x48.png
+      - type: file
+        path: 64x64.png
+      - type: file
+        path: 128x128.png
+      - type: file
+        path: 256x256.png
+      - type: file
+        path: 512x512.png
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - tar -xvzf n-track_flathub.tar.gz
+          - ARCH=$(uname -m)
+          - ln -s /usr/lib/${ARCH}-linux-gnu/libvulkan.so.1 lib/libvulkan.so
+          - rm *.tar.gz
+finish-args:
```

Comment:
this should be in the top

Reply:
there are about 2500 manifests lying around you in this repo, abundant resources to see how it works.